packages/jsii-pacmak/lib/targets/go/package.ts

```diff
@@ -107,6 +107,25 @@ export abstract class Package {
     this.emitTypes(context);
     code.closeFile(this.file);
 
+    if (this.types.length > 0) {
+      const initFile = this.file.replace(/(\.go)$/, '.init$1');
```

Comment:
Maybe use `path.basename(this.file, '.go')` instead?

Reply:
Duh I always forget this is a thing... -_-